x-pack/plugins/enterprise_search/public/applications/shared/deprecation_callout/deprecation_callout.tsx

```diff
@@ -15,85 +15,100 @@ import { docLinks } from '../doc_links';
 
 interface DeprecationCalloutProps {
   onDismissAction: () => void;
+  restrictWidth?: boolean;
 }
 
 export const EnterpriseSearchDeprecationCallout: React.FC<DeprecationCalloutProps> = ({
   onDismissAction,
+  restrictWidth = false,
 }) => {
+  const maxWidth = restrictWidth ? '75%' : '100%';
+  const cssStyles = {
+    maxWidth,
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    marginTop: '16px',
+    marginBottom: '16px',
+  };
   return (
-    <EuiCallOut
-      onDismiss={onDismissAction}
-      iconType={'warning'}
-      color={'warning'}
-      title={i18n.translate(
-        'xpack.enterpriseSearch.enterpriseSearchDeprecationCallout.euiCallOut.title',
-        { defaultMessage: 'Important Note' }
-      )}
-      data-test-subj="EnterpriseSearchDeprecationCallout"
-    >
-      <FormattedMessage
-        id="xpack.enterpriseSearch.deprecationCallout.first_message"
-        defaultMessage="The standalone App Search and Workplace Search products remain available in maintenance mode, and are not recommended for new search experiences. Instead, we recommend using our Elasticsearch-native tools which we are actively developing and improving, for your search use cases. These tools offer the flexibility and composability of working directly with Elasticsearch indices."
-      />
-      <EuiSpacer size="s" />
-      <FormattedMessage
-        id="xpack.enterpriseSearch.deprecationCallout.second_message"
-        defaultMessage="See this {workplaceSearchBlogUrl} for more information about upgrading your internal knowledge search or this {appSearchBlogUrl} about upgrading your catalog search."
-        values={{
-          workplaceSearchBlogUrl: (
-            <EuiLink
-              data-test-subj="workplaceSearch-deprecationCallout-blog-link"
-              href={docLinks.workplaceSearchEvolutionBlog}
+    <div style={cssStyles}>
+      <EuiCallOut
+        onDismiss={onDismissAction}
+        iconType={'warning'}
+        color={'warning'}
+        title={i18n.translate(
+          'xpack.enterpriseSearch.enterpriseSearchDeprecationCallout.euiCallOut.title',
+          { defaultMessage: 'Important Note' }
+        )}
+        data-test-subj="EnterpriseSearchDeprecationCallout"
+      >
+        <FormattedMessage
+          id="xpack.enterpriseSearch.deprecationCallout.first_message"
+          defaultMessage="The standalone App Search and Workplace Search products remain available in maintenance mode, but are not recommended for new search experiences. Instead, we recommend using our actively developed Elasticsearch-native tools. These tools offer the flexibility and composability of working directly with Elasticsearch indices."
```

Comment:
Oh, and I just saw that we're not mentioning Enterprise search
"The standalone **Enterprise Search product**, including App Search, and Workplace Search"
since the hosted web crawler will be a breaking change in 9.0, I think we should include Enterprise Search in the messaging